### PR TITLE
[Improvement] hivemq extension sdk version as property

### PR DESCRIPTION
**Pull request by AcidSepp**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ A new HiveMQ extension project can be created by running the following maven com
 
 [source,bash]
 ----
-mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.6.2
+mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DhivemqExtensionSdkVersion=4.6.3
```
Comment (SgtSilvio) [inline README.adoc]: we should probably still define archetypeVersion

**Review by SgtSilvio — APPROVED** [review]

File: README.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ A new HiveMQ extension project can be created by running the following maven com
 
 [source,bash]
 ----
-mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.6.2
+mvn archetype:generate -DarchetypeGroupId=com.hivemq -DarchetypeArtifactId=hivemq-extension-archetype -DarchetypeVersion=4.7.0 -DhivemqExtensionSdkVersion=4.6.2
```
Comment (DC2-DanielKrueger) [inline README.adoc]: i think we said in the standup that we can not make anything publish that has 4.7.0 in its name?

File: pom.xml [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
     <groupId>com.hivemq</groupId>
     <artifactId>hivemq-extension-archetype</artifactId>
-    <version>4.6.2</version>
+    <version>4.7.0</version>
```
Comment (DC2-DanielKrueger) [inline pom.xml]: sure? I think we said we use another version in the standup iirc

**Review by DC2-DanielKrueger — CHANGES_REQUESTED** [review]
version is not ok i think

**Review by DC2-DanielKrueger — APPROVED** [review]

**Review by SgtSilvio — APPROVED** [review]

**Pull request merged**